source/javascripts/_componentRegister.js

```diff
@@ -48,7 +48,7 @@ angular
 		"rWorkflowSelector",
 		register(WorkflowSelector, ["selectedWorkflow", "workflows", "selectWorkflow", "renameWorkflowConfirmed"])
 	)
-	.component("rWorkflowRecipesLink", register(WorkflowRecipesLink, ["id"]))
+	.component("rWorkflowRecipesLink", register(WorkflowRecipesLink, ["linkId"]))
```

Comment:
Is this used anywhere from angular?

Reply:
It might not be, it may be just muscle memory 😄